src/soap/handlers/GetSitesObject.js

```diff
@@ -19,12 +19,23 @@ import {
   soapBodyStart,
   soapBodyEnd,
   soapEnvelopeStart,
-  soapEnvelopeEnd
+  soapEnvelopeEnd,
+  soapHeaderStart,
+  soapWsaAction,
+  soapWsaMessageID,
+  soapWsaRelatesTo,
+  soapWsaTo,
+  soapWsuInfo,
+  soapWsseSecurityStart,
+  soapWsseSecurityEnd,
+  soapWsuTimestampStart,
+  soapWsuTimestampEnd,
+  soapHeaderEnd
 } from '../serializers/common.js'
 
 export async function* getSitesObject(
   request,
-  { date = new Date(), helpers, method, parameters }
+  { date = new Date(), helpers, method, parameters, uniqueid }
```

Comment:
Did we get clarification on if RelatesTo is even used or had relevance in this context? I can see you just used the same ID for both MessageID and RelatesTo, which I'm fine with. This is obviously not real SOAP, but rather some twisted mutation of it. Let's leave it for now and see how it goes. Seems you had to tweak the return XML. We just need to keep this in mind.